Power Up Proto/src/org/usfirst/frc/team5493/robot/OI.java

```diff
@@ -1,12 +1,13 @@
 package org.usfirst.frc.team5493.robot;
 import org.usfirst.frc.team5493.robot.commands.ClimbDown;
+//import org.usfirst.frc.team5493.robot.commands.ClimbHold;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)